### PR TITLE
ASUIS-269: Update asu_brand to manage the setting required for the new webdirectory

### PR DESCRIPTION
**Pull request by duarte-daniela**

### CODE REVIEW

File: asu_brand.install [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+/**
+ * Implements hook_update_N();
+ * Set search default settings.
+ */
+function asu_brand_update_9001() {
+  $config_factory = \Drupal::configFactory();
+  $config = $config_factory->getEditable('asu_brand.settings');
+  $config->set('asu_brand.asu_brand_search_url', 'https://search.asu.edu/search');
+  $config->set('su_brand.asu_brand_local_search_url', $GLOBALS['base_url']);
```
Comment (mlsamuelson) [inline asu_brand.install:11]: I think we have a typo... `su_brand` should be `asu_brand`.

Comment (duarte-daniela) [reply asu_brand.install:11]: @mlsamuelson you're right! I discovered the error when I runed the update in my local and I forgot to fix it here! thanks for catching i!

File: src/Form/AsuBrandSettingsForm.php [hunk 2]
```diff
@@ -91,6 +91,21 @@ public function buildForm(array $form, FormStateInterface $form_state) {
...
     ];
+    //Search settings
+    $form['asu_brand']['search_config'] = array(
+      '#type' => 'fieldset',
+      '#title' => $this->t('Search Settings'),
+    );
+    $form['asu_brand']['search_config']['asu_brand_search_url'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Search URL'),
+      '#default_value' => $config->get('asu_brand.asu_brand_search_url'),
```
Comment (mlsamuelson) [inline src/Form/AsuBrandSettingsForm.php:102]: Let's add a description: `URL to use for search in the header. Please don't update unless instructed. Format should match: https://search.asu.edu/search`

File: src/Form/AsuBrandSettingsForm.php [hunk 3]
```diff
@@ -91,6 +91,21 @@ public function buildForm(array $form, FormStateInterface $form_state) {
...
+    ];
+    $form['asu_brand']['search_config']['asu_brand_local_search_url'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Local Search URL'),
+      '#default_value' => $config->get('asu_brand.asu_brand_local_search_url'),
```
Comment (mlsamuelson) [inline src/Form/AsuBrandSettingsForm.php:107]: Add description: `If empty, your current site's base URL will be used. Optionally, you can override with the URL of your choice to be used for scoping local search. Use the format: https://yourdomain.asu.edu`

**Review by mlsamuelson — CHANGES_REQUESTED** [review]

**Review by mlsamuelson — APPROVED** [review]

**Pull request merged**